graphql/queries/templates.query.graphql

```diff
@@ -22,14 +22,18 @@ query Template($templateId: Int!) {
     created
     sections {
       id
-      displayOrder
       name
+      bestPractice
+      displayOrder
+      isDirty
       questions {
         errors
         displayOrder
         guidanceText
         id
         questionText
+        sectionId
```

Comment:
Do you need these here? They're returned above in the hierarchy. If it's more convenient to have them here, you can ignore this :)

Reply:
I think i added it because when I was using the Question type from the graphql-codegen-generated types file, it was complaining because I was missing sectionId.

I think for now I'll leave it there, but I could've just used my own interface I guess.

Thanks for reviewing this PR.